ms-common-impl/src/main/java/net/trajano/ms/vertx/jaxrs/JsonExceptionMapper.java

```diff
@@ -1,5 +1,8 @@
 package net.trajano.ms.vertx.jaxrs;
 
+import javax.ws.rs.ClientErrorException;
+import javax.ws.rs.InternalServerErrorException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.ws.rs.InternalServerErrorException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)